src/Sarif.FunctionalTests/SarifConverterTestData/FxCop/FxCopExampleReport2.xml.sarif

```diff
@@ -5,7 +5,22 @@
       "toolInfo": {
         "name": "FxCop"
       },
-      "runInfo": {},
+      "runInfo": {
+        "fileInfo": {
+          "d:\\SecDevTools\\Main\\bin\\x86\\Debug\\FxCop\\FxCopTestData\\Crypto\\SecurityCryptographyRuleTests.dll": [
```

Comment:
It's not intentional. I used OriginalString for the key. I see you have a helper `string.CreateUriForJsonSerialization`. I'll use that instead.